labo/zadanie1.c

```diff
@@ -0,0 +1,52 @@
+#include <stdio.h>
+
+int main() {
+
+  int a, b, c;
+
+  puts("podaj a");
+  scanf("%d", &a);
+
+  puts("podaj b");
+  scanf("%d", &b);
+
+  puts("podaj c");
+  scanf("%d", &c);
+
+
+  double wynik1 = a * b - ((double) b / c);
+  puts("wynik a)");
+  printf("%f\n", wynik1);
+
+  double wynik2 = a * b - (b % c);
+  puts("wynik b)");
+  printf("%f\n", wynik2);
+
+  double wynik3 = a * b - (c * a) - (b * c);
+  puts("wynik c)");
+  printf("%f\n", wynik3);
+
+
+
+  if ((a>b) && (b>c)) {
+    printf("%d, %d, %d\n",a,b,c);
+  }
+  else if((a>c) && (b>c) && (b>a))  {
```

Comment:
Jeżeli **a** jest większe od **c** oraz **b** jest większe od **a** to wiadomo, że **b** jest większe od **c**.
Środkowy warunek zbędny.